sym_api_client_python/clients/message_client.py

```diff
@@ -65,15 +65,24 @@ def _data_and_headers_for_attachment(self, stream_id, msg, filename, path_to_att
 
     def send_msg_with_attachment(self, stream_id, msg,
                                  filename, path_to_attachment):
```

Comment:
Maybe we should rename the parameter and add a hint like:
`def send_msg_with_attachment(self, stream_id, msg, filename, attachment: Union[str, io.BytesIO])`